src/main/java/software/amazon/smithy/lsp/SmithyInterface.java

```diff
@@ -54,13 +54,17 @@ public static Either<Exception, ValidatedResult<Model>> readModel(Collection<Fil
         // Contrary to model assemblers, model builders do not complain about the
         // duplication of shapes. Shapes will simply overwrite each other in a "last
         // write wins" kind of way.
-
+        // TODO It's not clear if we need ALLOW_UNKNOWN_TRAITS here
```

Comment:
Leaving a comment here to get thoughts on this

Reply:
I think allowing unknown traits here makes sense, especially given #73 since we know there's a need to allow pulling in other dependencies.